### PR TITLE
init produce graph

### PR DESCRIPTION
**Pull request by aguua**

### CODE REVIEW

File: graph/.gitignore [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.DS_Store
```
Comment (allgreed) [inline graph/.gitignore:1]: .

File: graph/Makefile [hunk 2]
```diff
@@ -1,11 +1,14 @@
 .DEFAULT_GOAL := help
+export FLASK_APP=src/main.py
+export FLASK_RUN_PORT=8000
+export FLASK_RUN_HOST=0.0.0.0
```
Comment (allgreed) [inline graph/Makefile:4]: Is this deliberate as in: https://www.gnu.org/software/make/manual/html_node/Variables_002fRecursion.html ?

File: graph/src/utils.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import json
+
+
+def get_json_content(json_path):
+    """Get json content.
```
Comment (allgreed) [inline graph/src/utils.py:5]: <3 enterprise much

File: graph/src/utils.py [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+    Args:
+        json_path (str): path to json.
+
+    Returns:
+        (dict): json content.
```
Comment (allgreed) [inline graph/src/utils.py:11]: Do you know [this](https://docs.python.org/3/library/typing.html) btw.?

**Review by allgreed — CHANGES_REQUESTED** [review]
Expect for the `.gitignore` looks good to me. I've added some questions and references, but mostly out of curiosity ;)

File: graph/src/utils.py (hunk 4 above)
Comment (aguua) [reply graph/src/utils.py:11]: si ;) enterprise +1

**Review by allgreed — APPROVED** [review]

**Pull request merged**